src/actplayer.cpp

```diff
@@ -4321,9 +4293,7 @@ void actPlayer(Entity* my)
 
 			if ( !command && !followerMenu.followerToCommand && followerMenu.recentEntity )
 			{
-				if ( *inputPressedForPlayer(PLAYER_NUM, impulses[IN_FOLLOWERMENU]) 
-					|| (inputs.bControllerInputPressed(PLAYER_NUM, INJOY_GAME_FOLLOWERMENU) 
-						&& players[PLAYER_NUM]->shootmode /*&& !players[PLAYER_NUM]->worldUI.bTooltipInView*/) )
+				if ( input.consumeBinaryToggle("Show NPC Commands") && players[PLAYER_NUM]->shootmode )
```

Comment:
I think clearing the binary toggle here is incorrect behavior. You can hold the button in, move to a follower wheel option, and then let go to execute the command

Reply:
Will fix